flake.nix

```diff
@@ -33,7 +33,27 @@
           cinaps = "*";
           ocamlfind = "1.9.2";
         };
-        pkgs = nixpkgs.legacyPackages.${system};
+        pkgs = nixpkgs.legacyPackages.${system}.appendOverlays [
+          (self: super: {
+            ocaml-ng = super.ocaml-ng // {
+              ocamlPackages_4_14 =
+                super.ocaml-ng.ocamlPackages_4_14.overrideScope' (oself: osuper: {
+                  melange-compiler-libs = with oself; buildDunePackage rec {
+                    pname = "melange-compiler-libs";
+                    version = "0.0.1-414";
+                    src = builtins.fetchurl {
+                      url = "https://github.com/melange-re/melange-compiler-libs/releases/download/${version}/melange-compiler-libs-${version}.tbz";
```

Comment:
Would it be possible to add melange-compiler-libs as a flake input instead?

Reply:
done in https://github.com/ocaml/dune/pull/6347/commits/76eab97a04049f68d63a902d4c7ecbdf590a7bd7